frontend/src/app/model/general-situation-daily.ts

```diff
@@ -0,0 +1,13 @@
+import {CovidCasesDaily} from './covid-cases-daily';
+import {CovidDeathsDaily} from './covid-deaths-daily';
+import {HospitalBedsDaily} from './hospital-beds-daily';
+
+export class GeneralSituationDaily {
```

Comment:
"activeCases" and "hospitalizations" does not exist in the json you get from the backend. It does not work when the names are not equal to them in the json from the backend.

Reply:
was gehört da ausgebessert im backend?

Reply:
Entweder du nennst die sachen im backend oder im frontend um aber sie Attribute der Objekte die du schickst müssen zusammenpassen.

Reply:
i weiß nit was du meinst sry :D

Reply:
Du hast die Variablen unten activeCases und hospitalizations genannt. Du musst die Variablen gleich nennen wie im backend in der klasse GeneralSituationPerDate

Reply:
is aber mittlerweile geändert